teams/sig_standard_cert/team_meetings.yml

```
@@ -89,3 +89,26 @@ events:
         - 2025-06-19 14:05:00  # public holiday in southern Germany, and others are gone as well
         - 2025-08-14 14:05:00  # holiday season; most people are on vacation
         - 2025-09-25 14:05:00  # SCS summit hangover, meeting of Forum SCS-Standards
+  - summary: "SIG Standardization/Certification"
+    begin: 2025-10-07 10:05:00
+    duration:
+      minutes: 50
+    description: |
+      In this Special Interest Group, we discuss and align our activities and approach to standardization and certification. That is to say, we devise and refine the relevant concepts and processes; we work on a roadmap for new certificate versions; and we align on which standards are desireable for each certificate subject. We then work with the teams to align on existing or new standards.
+      
+      Besides aspects of openness and sovereignty, the main goal of our standards is interoperability. We should take the user perspective: As a member of a DevOps team developing a service (think SaaS or PaaS) for SCS, I need XYZ. Every standard should be abstract enough to work regardless of the SCS reference implementation.
+      
+      This SIG is a joint effort by SCS and ALASCA.
+      
+      Minutes: https://github.com/SovereignCloudStack/minutes/tree/main/sig-standardization
+      Etherpad: https://input.scs.community/2025-scs-sig-standardization
+      Jitsi server on https://conf.scs.koeln:8443/SCS-Tech
+      Dial-In: +49-221-292772-611
+      Coordinator: Matthias Büchse <matthias.buechse[at]cloudandheat.com>
```

```suggestion
      Coordinator: Matthias Büchse <matthias.buechse[at]cloudandheat.com>
```

Shouldn’t this be your ALASCA Address?

---

Thx. Addressed with 8eeb273